dlt-database/entity/0002-refactor_add_community/Account.ts

```diff
@@ -8,8 +8,8 @@ import {
   BaseEntity,
 } from 'typeorm'
 import { User } from '../User'
-import { TransactionRecipe } from '../TransactionRecipe'
-import { ConfirmedTransaction } from '../ConfirmedTransaction'
+import { TransactionRecipe } from '../0001-init_db/TransactionRecipe'
+import { ConfirmedTransaction } from './ConfirmedTransaction'
```

Comment:
please add a comment above the code, why you make this change

Reply:
done